src/foxsisim/module.py

```diff
@@ -78,6 +78,22 @@ def __init__(self,
         else:
             self.shield = None
 
+        # sieve effect :
+        if thickness is None:
+            self.thickness = zeros(len(radii)).tolist()
+        elif isinstance(thickness, (tuple, list)) and len(thickness)==len(radii) :
+            self.thickness = thickness
+        else:
+            print('Error: Keyword thickness must have same length as radii')
+        self.sieveFront = []
+        self.sieveBack  = []
+        for thick, shell in zip(self.thickness, self.shells):
+            self.sieveFront.append(Annulus(center=base, normal=[0, 0, 1],
```

Comment:
I do not like the word sieve here. It is confusing and in a few years we may not understand what we meant.

Reply:
maybe call it shellAnnulus?